app/assets/templates/static_pages/index.html

```diff
@@ -46,8 +46,17 @@
       </ul>
     </li>
     <!-- logout -->
-    <li ng-if="loggedIn" ng-click="logout()" class="pull-right dropdown">
-      <a href="#/" class="dropdown-toggle" data-toggle="dropdown"><i class="fa fa-user"></i> Logout <b class=" glyphicon glyphicon-user"></b></a>
+    <li ng-if="loggedIn" class="pull-right dropdown">
```

Comment:
I would use `ng-show` instead of `ng-if`. Here is the difference: http://stackoverflow.com/questions/21869283/when-to-favor-ng-if-vs-ng-show-ng-hide